docs/learn/protocols/iota2.0/core-concepts/consensus/preliminaries.md

```diff
@@ -0,0 +1,288 @@
+# Preliminaries
+
+Before diving deeper into the consensus algorithm's inner workings, you should familiarize yourself with the following terms.
+
+## Epochs and Slots
+
+The IOTA protocol divides time into non-overlapping _slots_ of fixed duration `slotDurationSeconds`, set to `10` seconds. Slots are indexed with the slot index <code>$s = 0, 1, 2, 3,...$</code>. The slot indexed with $0$ is a unique slot that commences at negative infinity and concludes at the genesis timestamp `genesisUnixTime`. Every `2^slotsPerEpochExponent` sequential slots are grouped into _epochs_, which are enumerated with the epoch index <code>$e = 0, 1, 2, 3,...$</code>.
+
+Using slots primarily aims to generate [commitments](#slot-commitment-chain) for short time intervals. Generating commitments in a timely manner allows nodes to keep in Random Access Memory (RAM) only data of bounded size.
+
+Slot commitments improve the convergence of the overall consensus and let nodes know whether they need to synchronize their views on the Tangle. In addition, slot commitments facilitate data locality: understanding where a block belongs makes tasks such as block lookups, eviction, and pruning more straightforward. Furthermore, [finalization](consensus-flags.md#finalization-flag) occurs on the slot commitment level.
+
+The committee members are fixed within a given epoch, and their voting weights remain constant. However, at the epoch boundary, reconfiguration becomes possible, leading to potential changes in the composition of the committee and adjustments in their voting weights.
+
+### Epoch Committee
+
+For an epoch, a fixed subset of validators is responsible for achieving agreement about blocks and transactions issued during the epoch. This set has the size `committeeTotalSeats` ( between `25` and `50`). The selected set of validators is called the _epoch committee_.
+
+A committee member is expected to issue _validation blocks_ every `frequencyValidationBlock` seconds (set to either `0.5` or `1`) and to follow the [tip selection algorithm](tip-selection-algorithm.md) for selecting tips to be referenced by the validation blocks.
+
+#### Total committee
+
+Denote the total committee for epoch $e$ as <code>$\mathcal{C}_{total}(e)$</code>. Denote the _voting weight_ of a node $i$ at epoch $e$ as $W_i(e)$. Note that $W_i(e)>0$ if and only if $i$ is a committee member in epoch $e$, i.e., <code>$i\in \mathcal{C}_{total}(e)$</code>.
+
```

Comment:
:::info 

In the first version of the protocol, all committee members have the same weight, specifically if $W_i(e)>0$, then $W_i(e)=1$. This will be changed in future updates of the protocol.

:::

Reply:
```suggestion

:::info 

In the first version of the protocol, all committee members have the same weight, specifically if $W_i(e)>0$, then $W_i(e)=1$. This will be changed in future updates of the protocol.

:::

```